src/index.js

```diff
@@ -115,14 +115,16 @@ function configBuilder(addon_name, entrypoints) {
           loader: 'sass-loader',
           options: {
             additionalData:
-              `@import "@demos-europe/demosplan-ui/tokens/scss/_boxShadow.scss";
-              @import "@demos-europe/demosplan-ui/tokens/scss/_color.palette.scss";
-              @import "@demos-europe/demosplan-ui/tokens/scss/_color.data.scss";
-              @import "@demos-europe/demosplan-ui/tokens/scss/_color.brand.scss";
-              @import "@demos-europe/demosplan-ui/tokens/scss/_color.ui.scss";
-              @import "@demos-europe/demosplan-ui/tokens/scss/_fontSize.scss";
-              @import "@demos-europe/demosplan-ui/tokens/scss/_rounded.scss";
-              @import "@demos-europe/demosplan-ui/tokens/scss/_space.scss";`
+              `@import "@demos-europe/demosplan-ui/tokens/dist/scss/_boxShadow.scss";
```

Comment:
Why is the backtick here?

Reply:
this whole code block is being rendered as a javascript [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#syntax) into the css file.

Reply:
thx